calicoctl/commands/version.go

```diff
@@ -58,27 +61,18 @@ Description:
 	fmt.Println("Git commit:       ", GIT_REVISION)
 
 	// Load the client config and connect.
-	//cf := parsedArgs["--config"].(string)
-	//client, err := clientmgr.NewClient(cf)
-	//if err != nil {
-	//	fmt.Println(err)
-	//	os.Exit(1)
-	//}
-	//cfg := client.Config()
-	//a := apiconfig.NewCalicoAPIConfig()
-	//
-	//val, assigned, err := cfg.GetFelixConfig("CalicoVersion", "")
-	//if err != nil {
-	//	val = fmt.Sprintf("unknown (%s)", err)
-	//} else if !assigned {
-	//	val = "unknown"
-	//}
-	//fmt.Println("Cluster Version:  ", val)
-	//val, assigned, err = cfg.GetFelixConfig("ClusterType", "")
-	//if err != nil {
-	//	val = fmt.Sprintf("unknown (%s)", err)
-	//} else if !assigned {
-	//	val = "unknown"
-	//}
-	//fmt.Println("Cluster Type:     ", val)
+	cf := parsedArgs["--config"].(string)
+	client, err := clientmgr.NewClient(cf)
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	ctx := context.Background()
+	ci, err := client.ClusterInformation().Get(ctx, "default", options.GetOptions{})
+	if err != nil {
+		fmt.Println("Unable to retrieve Cluster Version or Type: ", err)
```

Comment:
I think we should os.Exit(1) here if we get an error getting the ClusterInfo. WDYT?

Reply:
It would be nice I guess to have that return code. I had purposefully not exited there because I was thinking about more code being added after but I think I'd rather get that return code than plan for something that might never happen.

Reply:
Yes, I think we should exit here with the right exit code, if we add something else later we can return the OR'd exit code later